content/content.js

```diff
@@ -164,3 +164,12 @@ chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
   messageHandler(message).then(sendResponse);
   return true;
 });
+
+/**
+ * Recieve messages
+ */
+chrome.runtime.onMessage.addListener(function (request, sender, sendResponse) {
+  if (request.action === "play") {
+    play();
+  }
+});
```

Comment:
This can be removed if we change the background script, there is a listener already